src/Table.luna

```diff
@@ -198,7 +533,23 @@ class Table:
         col2 = Column.fromList ("CORR_WITH_" + columnName) DoubleType vals
         Table.fromColumns [col1, col2]
 
-    # describeColumn :: Text -> Table 
+    # Generates descriptive statistics that summarize the central tendency, dispersion and shape of the values in given column distribution. 
+    #
+    # Analyzes just numeric values, excluding NA/null values. Returns statistics like: mean, standard deviation, minimum, 1st quantile, median, 3rd quantile, maximum.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > correlations = table.describeColumn "col2"
+    #
+    # `columnName`: String value, name of the column to calculate statistics for.
```

Comment:
What happens if a column with type other than numeric is specified?

Reply:
the statistics